### DISCUSSION TITLE
U1 Connection Issue after update

### DISCUSSION THREAD

**Discussion by rammelsberg** [NightscoutFoundation/xDrip#3510] [Q&A]
Hello!

My daughter has a bluejay U1 working perfectly for one week now. Today i did the system update and now i wont get any readings from the dexcom g6 sensor.
I already re-paired the transmitter (entering different code and changing back afterwards), i made some restarts, i unpaired the transmitter in the bluetooth list...
Now i'm stuck... what can i do next?
My daughter also has the Tandem T-Slim X2 pump, dont know if this is necessary information?!

In the status page of XDrip i can see a (good looking) last connected time of 3 mins currently but a "brain state" of "Power connect Errors:20"
XDRIP Version is c67b74f79-2024.05.28

I hope, we get it working again, tomorrow the school starts again, we need the follow function then...

Thank you!
Paul

**Comment by Navid200** [comment]
Has it had more than 15 minutes to scan for it without interruptions?
I mean without the transmitter ID changing or without the phone restarting etc.

Is there anything else connecting to the same transmitter other than t:slim like Dexcom app or another phone?

**Reply by rammelsberg** [reply]
I think, it had enough time when we had breakfast but to be sure, i will let the watch rest now in range of the sensor for half an hour.
No other device is connecting to the transmitter (just pump and U1 watch) the old watch (bluejay x2) was reconfigured to a different transmitter id when we switched to the new watch, i am sure it now ran out of battery (didnt charge since a week).

**Comment by Navid200** [comment]
Did you reboot the watch after updating software?

**Reply by rammelsberg** [reply]
yes, i did several restarts.

**Reply by Navid200** [reply]
If you look under the main xDrip menu on the watch, there is no start sensor option, is there?

**Reply by rammelsberg** [reply]
..No there is a "stop sensor" option.
If i open the bluetooth menu of android there currently is no paired device, before i changed something, there was a "dexcom 8?" device paired. maybe thats a problem? I thought, the pairing is done via xdrip when i enter a different trnsmitter id.
I did this already and "hunting for transmitter" message was shown in the system state, after some minutes the message disappeared.

**Comment by Navid200** [comment]
Please follow the instructions on the following link to enable detail logs.
https://navid200.github.io/xDrip/docs/Dexcom/OB1Debug.html

Let it be like that for 15 minutes while the watch is near G7.

Then, follow the instructions to upload.

**Reply by rammelsberg** [reply]
Hi!
I just uploaded the logs. 
I think i did quite a silly thing, while logging i went to bluetooth settings and saw a dexcom entry in the available devices and i clicked on it. Maybe there now are some confusing parts in the log. From the mac adress ist seems like the entry was from my wifes g6 not my daughters...

**Comment by Navid200** [comment]
How is the t:slim pump controlled?
Don't you need to run t:connect on an Android phone for that?
If your daughter already carries an Android phone to school, why not run xDrip on that same phone?

**Reply by rammelsberg** [reply]
Hi!
In Germany the t:connect is not yet available, the pump is controlled via its touch screen. 
We bought the bluejay watch so my daughter can always view her bg value without the need to pull out the pump or phone.
With the U1 watch, there now i no need to carry a phone at all, the watch has 4G connection to the internet. I will now temporarily switch back to the old watch+phone again because we need the follow function working tomorrow. But my goal is still to get the U1 working again.
I already tried to clear the bluetooth cache without success. 
Were you able to view the log files? 
Though i am an embedded sw developer i wasn't able to get useful information out of it...
Greetings and thank you for your help!
Paul

**Reply by Navid200** [reply]
The logs don't go to me.  They go to our lead developer.

**Comment by Navid200** [comment]
The t:slim pump uses the medical slot of the G6 transmitter.
As far as I know, U1 is also supposed to use the medical slot.  This could be the cause of a conflict.
